fdbrpc/include/fdbrpc/simulator.h

```diff
@@ -130,6 +130,7 @@ class ISimulator : public INetwork {
 	                              KillType kt,
 	                              KillType* newKillType) const = 0;
 	virtual bool isAvailable() const = 0;
+	virtual std::vector<AddressExclusion> getExcludeDCAddresses(Optional<Standalone<StringRef>> dcId) const = 0;
```

Comment:
How about `getAllAddressesInDCToExclude`? The current name feels like it only returns the already excluded addresses.

Reply:
Changed.